src/components/calcite-chip/calcite-chip.scss

```diff
@@ -94,18 +99,18 @@
     var(--calcite-chip-spacing-unit-s);
 }
 
-:host([dir="rtl"][icon]:not([dismissible])) span {
+:host([icon]:not([dismissible])) .calcite--rtl span {
   padding: var(--calcite-chip-spacing-unit-s) var(--calcite-chip-spacing-unit-s) var(--calcite-chip-spacing-unit-s)
     var(--calcite-chip-spacing-unit-l);
 }
 
 :host button {
-  @apply m-0 
-    p-0 
-    inline-flex 
-    self-stretch 
-    items-center 
-    border-none 
+  @apply m-0
+    p-0
```

Comment:
Should we remove `p-0` here, since padding is added on line 121?

It also looks like we can get the small size Chip's close button to not extend taller than the container by adding min/max height at 100%. What do you think?
```css
@apply m-0
    inline-flex
    self-stretch
    min-h-full /* new */
    max-h-full /* new */
    items-center
    border-none
    cursor-pointer
    bg-transparent
    duration-150
    ease-in-out
    text-color-1;
```

Reply:
Sounds good to me. 👍